docs/code_standards.rst

```diff
@@ -8,22 +8,25 @@ General
 
 In order to provide a code base that is easy to maintain, as well as easy to
 contribute to, **Robottelo** has adopted a set of code standards that all
-contributers are held to. Violations to our strictly held standards will result
-in a rejected pull request until all violations have been resolved. While not
-adhering to our recomended standards isn't a show stopper, keeping to them will
-help our code base to stay great!
+contributers are held to. Violations to our strictly held standards will result in a rejected pull request until all violations have been resolved. While not adhering to our recomended standards isn't a show stopper, keeping to them will help our code base to stay great!
 
 
 Strictly Held
 -------------
 
+Black
+
+* Robottelo uses the code formatting tool black.
```

Comment:
```suggestion
* Robottelo uses the code formatting tool: black.
```

Reply:
As its not a list, I disagree to the use of a colon. Some style guides suggest italics for the first use of a key term, or use quotation marks in a man mage.

Looking at https://github.com/psf/black I see the author uses a capital letter to write Black.

Reply:
OK. Whatever makes it more obvious it is a name of the tool.

Reply:
A link here would probably be beneficial to anyone new to the tool.

Reply:
@swadeley thoughts?

Reply:
Looking at the rendered guide[1], making the word "Black" a link to https://github.com/psf/black solves this issue as the word will be marked up in a blue font. No need for italics then.

[1] https://robottelo.readthedocs.io/en/latest/code_standards.html